app/retail/templates/shared/nav.html

```diff
@@ -20,12 +20,6 @@
   <a class="navbar-brand pt-0 pb-0" href="/bounties/funder">
     <img id="logo" src="{% static "v2/images/top-bar/bounties_desktop.svg" %}" height="35" alt="Logo" />
   </a>
-  {% if request.path|matches:"^\/((explorer)|(issue.*)|(.*new.*))$" %}
```

Comment:
we are not longer show this for metamask?

Reply:
it would be removed in  pages (where it's not needed like retail + explorer )
+ bounties (where we have the metamask checks which block user from taking any interaction on the page ) 
=====
It would still show up in other products 
AKA kudos, quests, grants, tips, ens (as the navbar code is duplicated there )